### PR TITLE
Started to address some comments from the sneak preview PR (EXPOSUREAPP-3456)

### PR DESCRIPTION
**Pull request by kolyaopahle**
### Description
This PR adresses some of the comments made in the #1643 PR

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -13,7 +13,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.0'
+        classpath 'com.android.tools.build:gradle:4.1.1'
```
Comment (chris-cwa) [inline build.gradle:16]: do we really want to update this in the feature branch? i would prefer to change this in its own branch, to (if something is not working anymore) we can decide to not yet update.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```
Comment (chris-cwa) [inline gradle/wrapper/gradle-wrapper.properties:3]: do we really want to update this in the feature branch? i would prefer to change this in its own branch, to (if something is not working anymore) we can decide to not yet update.

File: build.gradle (hunk 1 above)
Comment (kolyaopahle) [reply build.gradle]: no idea where this came from, removed it

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (kolyaopahle) [reply gradle/wrapper/gradle-wrapper.properties]: no idea where this came from, removed it

**Review by BMItr — APPROVED** [review]

**Pull request merged**